### PR TITLE
fix(DataTable): Add spacing to prevent clipping checkbox focus.

### PR DESCRIPTION
**Pull request by schillerk**
to: @milesj @stefhatcher @hayes @alecklandgraf

## Description
Before:
Screen Shot 2019-09-17 at 4 30 14 PM

After:
Screen Shot 2019-09-17 at 4 37 00 PM

## Checklist

- My code follows the style guide of this project.
- I have updated or added documentation accordingly.
- I have read the CONTRIBUTING document.

### CODE REVIEW

File: packages/core/src/components/DataTable/columns/renderSelectableColumn.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ export default function renderSelectableColumn(
     const spacing = isChild || !expandable ? indentSize : 0;
 
     return (
-      <Spacing left={spacing}>
+      <Spacing all={0.5} left={spacing}>
         <CheckBox
           label=""
```
Comment (stefhatcher) [inline packages/core/src/components/DataTable/columns/renderSelectableColumn.tsx]: wait a checkbox with no label or name?

**Comment by schillerk** [comment]
Got it thanks!

File: packages/core/src/components/DataTable/columns/renderSelectableColumn.tsx [hunk 2]
```diff
@@ -40,10 +40,10 @@ export default function renderSelectableColumn(
     const spacing = isChild || !expandable ? indentSize : 0;
 
     return (
-      <Spacing left={spacing}>
+      <Spacing all={0.5} left={spacing}>
         <CheckBox
-          label=""
-          name=""
+          hideLabel
+          label="Select row"
```
Comment (stefhatcher) [inline packages/core/src/components/DataTable/columns/renderSelectableColumn.tsx]: dope! does it need `T.phrase()` too?

Comment (schillerk) [reply packages/core/src/components/DataTable/columns/renderSelectableColumn.tsx]: ohhh yeah, done

**Pull request merged**